core/src/main/java/org/geysermc/geyser/util/SettingsUtils.java

```diff
@@ -126,6 +132,15 @@ public static CustomForm buildForm(GeyserSession session) {
                 if (session.getGeyser().getConfig().isAllowCustomSkulls()) {
                     session.getPreferencesCache().setPrefersCustomSkulls(response.next());
                 }
+
+                if (session.getGeyser().getConfig().isAboveBedrockNetherBuilding()) {
+                    boolean result = response.next();
+                    // If the setting changed, send a notice prompting the user of any side effects
+                    if (session.getPreferencesCache().isPrefersAboveBedrockNetherBuilding() != result) {
+                        session.getPreferencesCache().setPrefersAboveBedrockNetherBuilding(result);
+                        session.sendMessage("%geyser.settings.option.aboveBedrockNetherBuilding.notice");
```

Comment:
This needs to be `session.sendMessage(GeyserLocale.getLocaleString("geyser.settings.option.aboveBedrockNetherBuilding.notice", session.locale()))` or else it won't work.